app/views/helpers/TextGenerators.scala

```diff
@@ -21,12 +21,24 @@ import java.util.Locale
 import org.joda.time.LocalDate
 import org.joda.time.format.DateTimeFormat
 import play.api.data.Form
+import play.api.i18n.Messages
 import uk.gov.hmrc.time.TaxYear
-import views.helpers.WelshDateConverter._
+import utils.LanguageUtils.isWelsh
 
-object TextGenerators {
+object TextGenerator {
 
-  def separator(isWelsh: Boolean): String = if (!isWelsh) " to " else " i "
+  def apply()(implicit messages: Messages): TextGenerator = {
```

Comment:
suggestion refactor for the future we could have one TextGenerator that uses messages to determine the lang, and wouldnt require multiple implementations

Reply:
Yes I think this will be a ticket that needs creating